src/Pet.jsx

```diff
@@ -9,22 +9,23 @@
 //       React.createElement('h2', {}, props.breed),
 //     ]);
 //   };
+import { Link } from 'react-router-dom';
 
 const Pet = ({ name, animal, breed, images, location, id }) => {
   let hero = 'http://pets-images.dev-apis.com/pets/none.jpg';
   if (images.length) {
     hero = images[0];
   }
   return (
-    <a href={`/details/${id}`} className='pet'>
+    <Link href={`/details/${id}`} className='pet'>
```

Comment:
Save the href tags as constants in a separate file as a module. Save such things in the utils folder.